jobs/scripts/check_version.sh

```diff
@@ -111,17 +114,32 @@ for list in ${versionlist[@]}; do
 done
 
 nupkg_name="${pkgname}.${fullversion}.nupkg"
-feedlist=( "https://www.nuget.org/api/v2/package"
+feedlistv2=( "https://www.nuget.org/api/v2/package"
```

Comment:
Why was this renamed?

Reply:
It isn't necessary, I renamed it to distinguish from another `feedlist` that uses another web api.